locust/runners.py

```diff
@@ -747,32 +747,33 @@ def start(self, user_count: int, spawn_rate: float, wait=False) -> None:
         # when the user count is really at the desired value.
         timeout = gevent.Timeout(self._wait_for_workers_report_after_ramp_up())
         timeout.start()
+        msg_prefix = "All users spawned"
         try:
             while self.user_count != self.target_user_count:
                 gevent.sleep()
```

Comment:
We might want to change this to gevent.sleep(0.01) or something, to prevent busy waiting. What do you think @mboutet ?

Reply:
Makes sense